From c1c7dfea3de5857e840bc97c5d5c156f22e1e725 Mon Sep 17 00:00:00 2001
From: Hugo Mano <hugo@zml.ai>
Date: Tue, 16 Jun 2026 10:57:08 +0200
Subject: [PATCH 1/6] Drop the .so suffix for PjRt plugin targets (.dylib on
 macOS)

---
 xla/pjrt/c/BUILD | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xla/pjrt/c/BUILD b/xla/pjrt/c/BUILD
index 3f370f4f79..4675f05dd3 100644
--- a/xla/pjrt/c/BUILD
+++ b/xla/pjrt/c/BUILD
@@ -603,7 +603,7 @@ cc_library(
 
 # PJRT CPU plugin.
 xla_cc_binary(
-    name = "pjrt_c_api_cpu_plugin.so",
+    name = "pjrt_c_api_cpu_plugin",
     additional_linker_inputs = [
         ":pjrt_c_api_cpu_version_script.lds",
     ],
@@ -723,7 +723,7 @@ cc_library(
 
 # PJRT GPU plugin. Can be configured to be built for CUDA or ROCM.
 xla_cc_binary(
-    name = "pjrt_c_api_gpu_plugin.so",
+    name = "pjrt_c_api_gpu_plugin",
     additional_linker_inputs = [
         ":pjrt_c_api_gpu_version_script.lds",
     ],
-- 
2.50.1 (Apple Git-155)

