From 62241e96347b064dd01dd687ca86ac9b7a708f5d Mon Sep 17 00:00:00 2001
From: Hugo Mano <hugo@zml.ai>
Date: Wed, 5 Nov 2025 14:45:20 +0100
Subject: [PATCH 1/7] Drop the .so suffix for PjRt plugin targets (.dylib on
 macOS)

---
 xla/pjrt/c/BUILD | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xla/pjrt/c/BUILD b/xla/pjrt/c/BUILD
index e620ad271a..1fad146c11 100644
--- a/xla/pjrt/c/BUILD
+++ b/xla/pjrt/c/BUILD
@@ -545,7 +545,7 @@ cc_library(
 
 # PJRT CPU plugin.
 xla_cc_binary(
-    name = "pjrt_c_api_cpu_plugin.so",
+    name = "pjrt_c_api_cpu_plugin",
     additional_linker_inputs = [
         ":pjrt_c_api_cpu_version_script.lds",
     ],
@@ -662,7 +662,7 @@ cc_library(
 
 # PJRT GPU plugin. Can be configured to be built for CUDA or ROCM.
 xla_cc_binary(
-    name = "pjrt_c_api_gpu_plugin.so",
+    name = "pjrt_c_api_gpu_plugin",
     additional_linker_inputs = [
         ":pjrt_c_api_gpu_version_script.lds",
     ],
-- 
2.53.0

