From 5a073c6fc34b8ffeeb254bacdb1e8210c8cf6d62 Mon Sep 17 00:00:00 2001
From: Hugo Mano <hugo@zml.ai>
Date: Wed, 5 Nov 2025 14:45:51 +0100
Subject: [PATCH 2/3] Use hermetic cc toolchain for Linux CPU (use glibc 2.31)

---
 WORKSPACE | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)

diff --git a/WORKSPACE b/WORKSPACE
index 8d7f1384d8..8cd8df86a0 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -2,6 +2,7 @@
 workspace(name = "xla")
 
 load("//third_party:repo.bzl", "tf_http_archive", "tf_mirror_urls")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 # Initialize toolchains for ML projects.
 #
@@ -33,6 +34,28 @@ register_toolchains("@rules_ml_toolchain//cc:linux_aarch64_linux_aarch64")
 
 register_toolchains("@rules_ml_toolchain//cc:linux_aarch64_linux_aarch64_cuda")
 
+HERMETIC_CC_TOOLCHAIN_VERSION = "v3.1.1"
+
+http_archive(
+    name = "hermetic_cc_toolchain",
+    sha256 = "907745bf91555f77e8234c0b953371e6cac5ba715d1cf12ff641496dd1bce9d1",
+    urls = [
+        "https://mirror.bazel.build/github.com/uber/hermetic_cc_toolchain/releases/download/{0}/hermetic_cc_toolchain-{0}.tar.gz".format(HERMETIC_CC_TOOLCHAIN_VERSION),
+        "https://github.com/uber/hermetic_cc_toolchain/releases/download/{0}/hermetic_cc_toolchain-{0}.tar.gz".format(HERMETIC_CC_TOOLCHAIN_VERSION),
+    ],
+)
+
+load("@hermetic_cc_toolchain//toolchain:defs.bzl", zig_toolchains = "toolchains")
+
+# Plain zig_toolchains() will pick reasonable defaults. See
+# toolchain/defs.bzl:toolchains on how to change the Zig SDK version and
+# download URL.
+zig_toolchains()
+
+register_toolchains(
+    "@zig_sdk//toolchain:linux_amd64_gnu.2.31",
+)
+
 # Initialize the XLA repository and all dependencies.
 #
 # The cascade of load() statements and xla_workspace?() calls works around the
-- 
2.50.1 (Apple Git-155)
