diff --git a/xla/pjrt/c/BUILD b/xla/pjrt/c/BUILD
index aece9b02cb..28aa1d182e 100644
--- a/xla/pjrt/c/BUILD
+++ b/xla/pjrt/c/BUILD
@@ -410,7 +410,7 @@ xla_cc_binary(
     linkopts = if_macos(
         [
             "-Wl,-exported_symbol,_GetPjrtApi",
-            "-Wl,-install_name,@rpath/pjrt_c_api_cpu_plugin.dylib",
+            "-Wl,-install_name,@rpath/libpjrt_cpu.dylib",
             "-Wl,-undefined,error",
         ],
         [
---
