From 9e5012872821846f74bd7600eefb7c99218fc6ee Mon Sep 17 00:00:00 2001
From: Corentin Kerisit <corentin.kerisit@gmail.com>
Date: Tue, 16 Jun 2026 10:57:08 +0200
Subject: [PATCH 2/6] Custom install name

---
 xla/pjrt/c/BUILD | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xla/pjrt/c/BUILD b/xla/pjrt/c/BUILD
index 4675f05dd3..862ed96c42 100644
--- a/xla/pjrt/c/BUILD
+++ b/xla/pjrt/c/BUILD
@@ -610,7 +610,7 @@ xla_cc_binary(
     linkopts = if_macos(
         [
             "-Wl,-exported_symbol,_GetPjrtApi",
-            "-Wl,-install_name,@rpath/pjrt_c_api_cpu_plugin.dylib",
+            "-Wl,-install_name,@rpath/libpjrt_cpu.dylib",
             "-Wl,-undefined,error",
         ],
         [
-- 
2.50.1 (Apple Git-155)

