From 0ffb2a6b53aaadd49348e706afb2c39af2ac5b4b Mon Sep 17 00:00:00 2001
From: Hugo Mano <hugo@zml.ai>
Date: Mon, 21 Jul 2025 14:38:20 +0200
Subject: [PATCH] cuda: use hermetic

---
 tensorflow.bazelrc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tensorflow.bazelrc b/tensorflow.bazelrc
index 05b4f8af3f..cde557c7f5 100644
--- a/tensorflow.bazelrc
+++ b/tensorflow.bazelrc
@@ -196,7 +196,7 @@ build:cuda --config=cuda_version
 # This flag is needed to include CUDA libraries.
 build:cuda --@local_config_cuda//cuda:include_cuda_libs=true
 # TODO(ybaturina): Remove this once the wheel size issue is fixed.
-build:cuda --config=clang_local
+# build:cuda --config=clang_local
 
 # This configuration is used for building the wheels.
 build:cuda_wheel --@local_config_cuda//cuda:include_cuda_libs=false
@@ -256,7 +256,7 @@ build:asan --copt -g
 build:asan --copt -fno-omit-frame-pointer
 build:asan --linkopt -fsanitize=address
 
-build:rocm_base --config=clang_local
+# build:rocm_base --config=clang_local
 build:rocm_base --copt=-Wno-gnu-offsetof-extensions
 build:rocm_base --crosstool_top=@local_config_rocm//crosstool:toolchain
 build:rocm_base --define=using_rocm_hipcc=true
-- 
2.39.5 (Apple Git-154)

