From 023c27c0d58f42898ac0f6dd5cedbd72a7597659 Mon Sep 17 00:00:00 2001
From: Hugo Mano <hugo@zml.ai>
Date: Mon, 21 Jul 2025 14:38:20 +0200
Subject: [PATCH] cuda: use hermetic

---
 tensorflow.bazelrc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tensorflow.bazelrc b/tensorflow.bazelrc
index 05b4f8af3f..56039cb37f 100644
--- a/tensorflow.bazelrc
+++ b/tensorflow.bazelrc
@@ -196,7 +196,7 @@ build:cuda --config=cuda_version
 # This flag is needed to include CUDA libraries.
 build:cuda --@local_config_cuda//cuda:include_cuda_libs=true
 # TODO(ybaturina): Remove this once the wheel size issue is fixed.
-build:cuda --config=clang_local
+# build:cuda --config=clang_local
 
 # This configuration is used for building the wheels.
 build:cuda_wheel --@local_config_cuda//cuda:include_cuda_libs=false
-- 
2.39.5 (Apple Git-154)

