From 88d7974c0d2eca06d05ffeebdd512343119deb34 Mon Sep 17 00:00:00 2001
From: Hugo Mano <hugo@zml.ai>
Date: Thu, 10 Apr 2025 13:56:17 +0200
Subject: [PATCH 8/8] deps: triton integrate branch 1.3

---
 third_party/triton/workspace.bzl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/triton/workspace.bzl b/third_party/triton/workspace.bzl
index 656679a89d..895c7b4c96 100644
--- a/third_party/triton/workspace.bzl
+++ b/third_party/triton/workspace.bzl
@@ -8,8 +8,8 @@ load("//third_party/triton:xla_extensions/series.bzl", "extensions_files_patch_l
 def repo():
     """Imports Triton."""
 
-    TRITON_COMMIT = "triton_integrate_branch-1.2"
-    TRITON_SHA256 = "ba715575f8e8ead49df545a40c9557a4e40174400892fcf28fefdd15ff3f2c6a"
+    TRITON_COMMIT = "triton_integrate_branch-1.3"
+    TRITON_SHA256 = "930d2c40ded0300c4070b8f43a845d493c7a078a4864d023be3af2e16ec4f884"
     tf_http_archive(
         name = "triton",
         sha256 = TRITON_SHA256,
-- 
2.39.5 (Apple Git-154)

